tests/integration/test_definitions/s3/select_batch_by_path.py

```diff
@@ -0,0 +1,27 @@
+import great_expectations as gx
+
+context = gx.get_context()
+
+datasource_name = "my_s3_datasource"
+bucket_name = "superconductive-docs-test"
+
+datasource = context.data_sources.add_pandas_s3(
+    name=datasource_name, bucket=bucket_name, boto3_options={}
+)
+
+asset_name = "my_taxi_data_asset"
+s3_prefix = "data/taxi_yellow_tripdata_samples/"
+data_asset = datasource.add_csv_asset(name=asset_name, s3_prefix=s3_prefix)
+
+batch_definition = data_asset.add_batch_definition_path(
+    "monthly",
```

Comment:
Just for clarity, I think we should name this something different:
```suggestion
    "filepath",
```

Reply:
Good call. Fixed!